api/fs/dir.go

```diff
@@ -36,12 +36,11 @@ func (d *dir) List(ctx context.Context) ([]plugin.Entry, error) {
 }
 
 func (d *dir) ChildSchemas() []*plugin.EntrySchema {
-	return []*plugin.EntrySchema{
-		d.Schema(),
-		(&file{fsnode: &fsnode{}}).Schema(),
-	}
+	return nil
 }
 
 func (d *dir) Schema() *plugin.EntrySchema {
```

Comment:
I wonder if we should just remove this package -- the shell experience cds to the mountpoint anyways so in the common case, users will not touch local files/dirs.

If we still want things to work with local files/dirs, I can file an issue to move this code over to a client wrapper.

Reply:
I think the goal is for the shell experience to feel contained in that. I have issues up to display the path starting from the Wash root, and for `cd` to take you back to the Wash root. So I think it should be fine to error if you try to use Wash tools outside that.

Reply:
Ok, think a separate "Remove the apifs package" issue would make sense?

Reply:
Sure, mostly as tech-debt.